src/InstrumentationEngine.Lib/LoggerService.cpp

```diff
@@ -304,18 +333,23 @@ HRESULT CLoggerService::RecalculateLoggingFlags()
     {
         // Reset the sink; get its desired logging level
         LoggingFlags sinkFlags;
+        LoggingFlags instruMethodSinkFlags;
         IfFailRetNoLog(pSink->Reset(m_defaultFlags, &sinkFlags));
+        IfFailRetNoLog(pSink->Reset(m_allInstruMethodFlags, &instruMethodSinkFlags));
 
         // Add the sink to each logging level vector
-        if (IsFlagSet(sinkFlags, LoggingFlags_Errors))
+        if (IsFlagSet(sinkFlags, LoggingFlags_Errors) ||
+            IsFlagSet(instruMethodSinkFlags, LoggingFlags_Errors))
```

Comment:
can this be done similar to above:

IsFlagSet(sinkFlags, | instruMethodSinkFlags, LoggingFlags_Errors)